lib/swaggerValidator/trafficValidator.ts

```diff
@@ -158,12 +163,22 @@ export class TrafficValidator {
 
   public async validate(): Promise<TrafficValidationIssue[]> {
     let payloadFilePath;
+    const swaggerOpts: SwaggerLoaderOption = {
+      setFilePath: false,
+    };
+    this.swaggerLoader = inversifyGetInstance(SwaggerLoader, swaggerOpts);
+    const spec = await this.swaggerLoader.load(this.specPath);
```

Comment:
`specPath` could be a folder, use `swaggerFile` in `for` loop.

Reply:
deleted